supply-chain/audits.toml

```diff
@@ -343,6 +377,11 @@ The only changes from 0.6.1 were from my own PR which updated memfd to newer
 dependencies.
 """
 
+[[audits.memoffset]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+delta = "0.7.1 -> 0.8.0"
```

Comment:
Like the above updates it'd be good to add a `notes` here (the changes all look fine to me at least in the diff)

Reply:
Done